charts/consul/templates/terminating-gateways-deployment.yaml

```diff
@@ -66,7 +65,13 @@ spec:
         release: {{ $root.Release.Name }}
         component: terminating-gateway
         terminating-gateway-name: {{ template "consul.fullname" $root }}-{{ .name }}
+        consul.hashicorp.com/connect-inject-managed-by: consul-k8s-endpoints-controller
       annotations:
+        "consul.hashicorp.com/connect-inject": "false"
+        "consul.hashicorp.com/gateway-kind": "terminating"
+        {{- if $root.Values.global.enableConsulNamespaces }}
+        "consul.hashicorp.com/gateway-namespace": {{ (default $defaults.consulNamespace .consulNamespace) }}
```

Comment:
Originally, I called this `terminating-gateway-namespace`. But I've changed it to `gateway-namespace` so that it may also be used for Ingress gateways when we get to them. My goal here was to reduce the overall number of different annotations by... one. :shrug: